kubernetes/schema_container.go

```diff
@@ -322,7 +322,7 @@ func containerFields(isUpdatable, isInitContainer bool) map[string]*schema.Schem
 		"env_from": {
 			Type:        schema.TypeList,
 			Optional:    true,
-			ForceNew:    true,
+			ForceNew:    !isUpdatable,
```

Comment:
Looks like this line should just be removed as it is covered by https://github.com/terraform-providers/terraform-provider-kubernetes/blob/97f7696e7fc3e6e93b36706f85525217ff19ff15/kubernetes/schema_container.go#L539